source/_integrations/sharkiq.markdown

```diff
@@ -21,21 +20,78 @@ The `sharkiq` integration allows you to control your [Shark IQ](https://www.shar
 <p class='img'>
 <img src='/images/screenshots/more-info-dialog-sharkiq.png' />
 </p>
-
-This platform has been tested and is confirmed to be working with the Shark IQ R101AE robot vacuum with self-empty base but should also work with the R100.
+<div class='note'>
+This integration uses your device's cloud API to communicate with the device.  It has been tested with the R101AE and
+AV2001 robots, but should be compatible with all smart SharkIQ robot vacuums.  If you are having issues with your device not
```

Comment:
Lets adds a chapter with reported working devices. We don't need a big note box for that.